README.md

```diff
@@ -21,21 +21,16 @@ Install
 
 Requires [PHP CodeSniffer](http://pear.php.net/package/PHP_CodeSniffer/) version 3.x with PHP 5.4 or higher.
 
-Because of the way PHP CodeSniffer works, you need to put the `Security/` folder from phpcs-security-audit in `/usr/share/php/PHP/CodeSniffer/Standards` or do a symlink to it.
-
-The easiest way to install is to git clone and use composer that will create the symlink for you:
-```
-composer install
-./vendor/bin/phpcs --standard=example_base_ruleset.xml tests.php
-```
-
-The package is also on [Packagist](https://packagist.org/packages/pheromone/phpcs-security-audit):
```

Comment:
what is the reason not to link to packagist anymore?

Reply:
Well, the original install instructions were based on `git clone` with a Composer based install being mentioned as the alternative, which is were the link came in.

This has now been reversed, with the regular install instructions now being to use Composer and the alternative being to use `git clone`.

As the install instructions now mention doing a `composer require` for the package:
1. There is no real reason to have an explicit link to Packagist anymore.
2. The fact that the package is on Packagist is implied.
3. No typical text left over which to link to Packagist.

Happy to bring it back, but we'd need to think of some text to link to it then.

Reply:
Yeah I might need the link to remind myself this is where the release goes 😅. I remember having to do something on it a few years when we changed from pheromone to FloeDesignTechnologies.

Reply:
Well, if it's just for _you_, then there's no reason to bring the link back: if you go to the Packagist website and login, there is a `My Packages` link in the drop-down under your user name.
Also: https://packagist.org/users/jmarcil/packages/

Reply:
I'm afraid you overestimate my abilities to remember something every 5 years ;-)

I won't even remember packagist is a thing in two weeks.

Reply:
*grin* What about adding some badges to the top of the `README` with project status information ? Those (in part) would then link to Packagist.